requirements.in

```diff
@@ -1,5 +1,5 @@
-astroplan
-astropy
+git+git://github.com/astropy/astroplan@master#egg=astroplan
```

Comment:
I assume the plan would be to revert to a pip install once the astroplan on PyPI is new enough to include the necessary fixes?

Reply:
Yes, and thanks for the reminder, I'll create an issue.